Made a small change to IIS Application - Create

gislikonrad opened the pull request:
Added ability to convert a sub directory of an iis website into an iis application

alfhenrik commented:
Hi @gislikonrad,

Thanks for your contribution to the library!

One little thing, instead of commenting out the `throw` statement, could we perhaps write to output that the physical path of the virtual directory does not exist and will be converted an IIS application, just to provide some feedback on what is happening?

Other than that, it all looks good to me.

Thanks,
Henrik

gislikonrad commented:
Sorry I took so long to respond. Just got back from vacation. :)

I fixed a bug in the script and added more verbose messages to my changes (as requested)

alfhenrik commented:
Thanks very much for your contribution, this has been included in v1.215 which has been released to production now.
